bin/gwdetchar-lasso-correlation

```diff
@@ -85,8 +85,10 @@ parser.add_argument('-p', '--primary-channel',
 parser.add_argument('-P', '--primary-frametype', default=None,
                     help='frametype for --primary-channel, default: guess '
                          'by channel name')
-parser.add_argument('-O', '--remove-outliers', type=float, default=None,
+parser.add_argument('-O', '--remove-outliers-s', type=float, default=None,
```

Comment:
I suggest we keep the original name of this argument, so that end users don't have to change any script invocations